### PR TITLE
Add CRC32 at the end of a transfer and demark the payload as a field

### LINKED ISSUE #5: Add CRC32 at the end of a transfer and demark the payload as a field
The CRC32 should be a field plus the payload as a byte array.

### PR DESCRIPTION
**Pull request by emrainey**
Fixes #5
Added Payload size too and fixed names for CRC.

### CODE REVIEW

File: cyphal_udp.lua
Comment (thirtytwobits) [inline cyphal_udp.lua:1]: The version is only 4 bits. The following 4 bits are reserved.

File: cyphal_udp.lua [hunk 1]
```diff
@@ -23,7 +23,10 @@ frame_index_eot = ProtoField.uint32("cyphal_udp.frame_index_eot", "frame_index_e
 frame_index = ProtoField.uint32("cyphal_udp.frame_index", "frame_index", base.DEC)
 end_of_transfer = ProtoField.bool("cyphal_udp.end_of_transfer", "end_of_transfer", base.NONE)
 user_data = ProtoField.uint16("cyphal_udp.user_data", "user_data", base.HEX)
-crc16_ccitt_false = ProtoField.uint16("cyphal_udp.crc16_ccitt_false", "crc16_ccitt_false (BE)", base.HEX)
+crc16_ccitt_false = ProtoField.uint16("cyphal_udp.crc16_ccitt_false", "CRC16-CCITT-FALSE (BE)", base.HEX)
```
Comment (thirtytwobits) [inline cyphal_udp.lua:26]: It's all upper-case now. Okay. Am I supposed to be impressed?

**Review by thirtytwobits — APPROVED** [review]
You'd have to help me with the APIs for my review to be really worth-while but as long as this matches the specification (https://opencyphal.org/specification/Cyphal_Specification.pdf) then we're good.

File: cyphal_udp.lua (hunk 1 above)
Comment (emrainey) [reply cyphal_udp.lua:26]: All the reference online are in SHOUTY CASE.  So I thought I'd SHOUT too.

**Pull request merged**